### PR TITLE
Add scRNA Pre-Processing hands-on

### PR DESCRIPTION
**Pull request by mtekman**

### CODE REVIEW

**Comment by mtekman** [comment]
@bgruening thanks

**Comment by mtekman** [comment]
This still requires:
 
* Single-Batch/Multi-Batch name changes in SVG file
* Better Upper and Lower workflow images (no red boxes, use fade instead)
* Less wordy intro
* 50% image class
* Add this as follow_up to "Understanding Barcodes"

**Comment by mtekman** [comment]
Please review, made the required changes.

File: topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md [hunk 1]
```diff
@@ -148,19 +146,17 @@ The size of scRNA files (.fastq) are typically in the gigabyte range and are som
 {: .hands_on}
 
 
-For a more detailed understanding of the naming conventions used in generating our FASTQ pair, please consult the [*Plates, Batches, and Barcodes*](../scrna-plates-batches-barcodes/slides.html) slides.
+For a more detailed understanding of the naming conventions used in generating our FASTQ pair, please consult the [*Plates, Batches, and Barcodes*]({{ site.baseurl }} topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html %}) slides.
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md]: is there a / missing and a whitespace to much?

Comment (mtekman) [reply topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md]: not seeing it, where is the whitespace?

**Comment by mtekman** [comment]
@bebatut thanks for the review!

File: topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md (hunk 1 above)
Comment (bgruening) [reply topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md]: That was a comment for @bebatut. Shoudn't it be `{{ site.baseurl }}/topics/transcriptomics/tutorials/s...`

Comment (nsoranzo) [reply topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md]:
```suggestion
For a more detailed understanding of the naming conventions used in generating our FASTQ pair, please consult the [*Plates, Batches, and Barcodes*]({{ site.baseurl }}{% link topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html %}) slides.
```

Comment (bebatut) [reply topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md]: I aldready fixed it in my last commit. Thanks @bgruening and @nsoranzo for catching it

**Comment by bgruening** [comment]
Awesome! Thanks @mtekman!

**Pull request merged**